src/rand/mod.rs

```diff
@@ -1,3 +1,18 @@
 //! Pseudo-random element generation.
 
 pub use winter_crypto::{RandomCoin, RandomCoinError};
+
+use crate::{Felt, Word, ZERO};
+
+mod rpo;
+
+/// Pseudo-random basefield elements generator.
+///
+/// An instance can be used to draw, uniformly at random, basefield elements as well as `Word`s.
+pub trait RandFeltsGen {
```

Comment:
Not sure if I like `RandFeltsGen` better than `Rng`. Should we just go with `Rng`? Or maybe `FeltRng`?

Also minor nit: on line 9 I'd probably say just: "Pseudo-random element generator."

Reply:
Makes sense! Updated to `FeltRng` and changed the comment.